src/api/app/assets/stylesheets/webui/application.scss

```diff
@@ -26,6 +26,7 @@
 @import 'datatables';
 @import 'card-component';
 @import 'breadcrumbs-component';
+@import 'tabs';
```

Comment:
What do you think of merging `tabs` and `tabs-component`? I don't think that we need to have two files for this since they are both for the same component.

Reply:
Done!